cmd/main.go

```diff
@@ -336,6 +339,14 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "ManagementBackup")
 		os.Exit(1)
 	}
+	if err = (&ipam.ClusterIPAMClaimReconciler{
```

Comment:
should have ability to be turned on and off via CLI args

Reply:
flag should also be in chart with corresponding `values` entry to override default value via helm